userland/init/init.c

```diff
@@ -23,6 +23,12 @@ void uptime() {
     printf("implement me\n");
 }
 
+int overflow() {
+    char c[12];
+    strcpy(c, "123456789012345678901234567890");
+    return 1;
```

Comment:
There and in the other function, I don't think this `return` statement is really needed.

Reply:
We need the return because especially non-void functions that return a value will be affected by this. Even if it's not used.

Reply:
I can check if it works with a void too tho.